bin/install.sh

```diff
@@ -398,8 +398,12 @@ function _compile_libuv {
 
 function _check_compiler_features {
   echo "# checking compiler features"
-  local cflags=($("$root/bin/cflags.sh" -Os))
-  $CXX -x c++ "${cflags[@]}" -o /dev/null - >/dev/null << EOF_CC
+  local cflags=($("$root/bin/cflags.sh"))
+  if [[ -z "$DEBUG" ]]; then
+    cflags+=(-o /dev/null)
+  fi
```

Comment:
Had to omit `-o /dev/null` when `-g` exists in cflags. 

Otherwise you get this error:
```
❯ bin/install.sh
# checking compiler features
error: cannot parse the debug map for '/dev/null': The file was not recognized as a valid object file
clang: error: dsymutil command failed with exit code 1 (use -v to see invocation)
not ok - /usr/bin/clang++ (14.0.0) clang > 11 is required for building socket - please report (https://discord.gg/YPV32gKCsH)
```